cico_setup.sh

```diff
@@ -53,7 +53,7 @@ function run_functional_tests() {
 
     # Run the E2E tests against the running fabric8-ui container
     docker exec -t -e REFRESH_TOKEN=$REFRESH_TOKEN $CID bash -c \
-        "cd tests && WEBDRIVER_VERSION=2.37 DEBUG=true HEADLESS_MODE=true BASE_URL='https://prod-preview.openshift.io/' ./run_e2e_tests.sh"
+        "cd tests && WEBDRIVER_VERSION=2.37 DEBUG=true HEADLESS_MODE=true BASE_URL='http://${SERVER_IP}:8080' ./run_e2e_tests.sh"
```

Comment:
LGTM

Reply:
@rgarg1 this was the temporary change to run tests against prod-preview, I changed it back.
We will be using cico_test_prod.sh to tests against production